scripts/make-version.js

```diff
@@ -61,17 +61,17 @@ version = '${version}'
             `Version file for pip version ${version} saved sucessfully.`);
       });
 
-  const buildFilename = dirName + '/python/BUILD';
+  const buildFilename = dirName + '/python/BUILD.bazel';
```

Comment:
nit: os.path.join ?

Reply:
Good idea. Fixed all instances in the script.

Reply:
my bad - I thought it's a python file. Thanks for the change!